common/k8stest/util_rdma.go

```diff
@@ -317,3 +317,41 @@ func GetDevLinkName(node, iface string) (string, error) {
 	}
 	return "", nil
 }
+
+func DisableRdmaDeicesOnAllMayastorNodes() error {
```

Comment:
typo ` DisableRdmaDeicesOnAllMayastorNodes` -> ` DisableRdmaDevicesOnAllMayastorNodes`

Reply:
Not great, but would this be better named as `DisableConfiguredDisabledRdmaDevicesOnAllMayastorNodes` ?

Reply:
Updated the function name https://github.com/openebs/openebs-e2e/compare/97f6bb9f1285bb8f08fe736bb498948de52c0b3f..7a4e8d1886a4b1bebeba350d30dde9c35e2ec096